recipes/sng/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+./configure --prefix="${PREFIX}" --with-rgbtxt="${PREFIX}/share/netpbm/rgb.txt"
+make
+make install
```

Comment:
Is there a make check we can run here?

Reply:
Added: required a semi-icky `http` download of [pngsuite](http://www.schaik.com/pngsuite/):

```
http://www.schaik.com/pngsuite/PngSuite-2017jul19.tgz
```

I guess this could be:
- its own `noarch: generic` package?
- downloaded from a trustworthy snapshot service, e.g. WayBack machine: 
```
https://web.archive.org/web/20230708171113/http://www.schaik.com/pngsuite/PngSuite-2017jul19.tgz
```

Reply:
This is fine for now. But if you want to make is a separate noarch generic package later, go for it.
I'm not sure what else would need it to make it worth it.

Reply:
Well, schlepping `diffoscope` deps has me thinking about Reproducibility a lot :blush: